cli/lib/test/png2src.test.js

```diff
@@ -165,11 +165,11 @@ describe("PNG2SRC", () => {
     });
 
     it("should abort and output an error message for invalid image paths", () => {
-      runAll(["/invalid/path"], {
+      expect(() => 
+        runAll(["/invalid/path"], {
         lang: "rust",
         output: "-",
-      });
-      expect(console.error).toHaveBeenCalledWith("Error processing /invalid/path: ENOENT: no such file or directory, open '/invalid/path'");
+      })).toThrowError(/Error processing \/invalid\/path: ENOENT: no such file or directory, open '\/invalid\/path'/);
```

Comment:
@peppergrayxyz I think it could be worth asserting the process exits with code error 1 with something like
```
const mockExit = jest.spyOn(process, 'exit').mockImplementation(() => {});
// expect with runAll() call here then
expect(mockExit).toHaveBeenCalledWith(1);
```
WDYT?

Reply:
I tried to fit it into the existing logic and `run` in `png2src.js` does not return a value either, but only throws an exception, so I though `runAll` should behave the same way. Also, the other modules don't return a value either, but throw exceptions. So I think an exception is the most consistent thing to do here.

If we want to check for `return 1` I think we should do it in the test for `cli.js` and then also for all such cases, but there are no tests for that file (yet).

Reply:
Okay, I don't know this tool yet, you are perfectly right. Thanks for your detailed answer.